app/blockchain/fabric-samples/install-fabric.sh

```
@@ -0,0 +1,377 @@
+fi
+
+# Prior to fabric 2.5, use amd64 binaries on darwin-arm64
+if [[ $VERSION =~ ^2\.[0-4]\.* ]]; then
+  PLATFORM=$(echo $PLATFORM | sed 's/darwin-arm64/darwin-amd64/g')
+fi
+
+BINARY_FILE=hyperledger-fabric-${PLATFORM}-${VERSION}.tar.gz
+CA_BINARY_FILE=hyperledger-fabric-ca-${PLATFORM}-${CA_VERSION}.tar.gz
+
+# if nothing has been specified, assume everything
+if [[ ${_arg_comp[@]} =~ ^$ ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)